src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -110,77 +86,31 @@ protected bool InitializeFileWatcherIfEnabled()
 
         public async Task Invoke(object[] parameters)
         {
+            FunctionInvocationContext context = GetContextFromParameters(parameters, Metadata);
+            await InvokeCore(parameters, context);
+        }
+
+        private static FunctionInvocationContext GetContextFromParameters(object[] parameters, FunctionMetadata metadata)
```

Comment:
pre-existing class., we can rename separately

---
In reply to: [126829107](https://github.com/Azure/azure-webjobs-sdk-script/pull/1660#discussion_r126829107) [](ancestors = 126829107)